financial-services-accelerator/react-apps/self-care-portal/src/main/java/org/wso2/financial/services/accelerator/scp/webapp/servlet/ResourceInterceptorServlet.java

```diff
@@ -112,7 +118,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) {
                     // add existing req headers to new request
                     Map<String, String> headers = Collections.list(req.getHeaderNames())
                             .stream()
-                            .filter(h -> !HttpHeaders.AUTHORIZATION.equalsIgnoreCase(h))
+                            .filter(h -> !HttpHeaders.AUTHORIZATION.equals(h))
```

Comment:
Shall we make the headers capital?

Reply:
FIxed in https://github.com/wso2/financial-services-accelerator/pull/752/commits/1e2edab95303b5a528064f8a4ceb3acb3d75fd36